Makefile

```diff
@@ -356,7 +356,7 @@ managers: ## Alias for manager-aws-infrastructure
 
 .PHONY: manager-aws-infrastructure
 manager-aws-infrastructure: ## Build manager binary
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "${LDFLAGS} -extldflags '-static'" -o $(BIN_DIR)/manager .
+	CGO_ENABLED=0 go build -ldflags "${LDFLAGS} -extldflags '-static'" -o $(BIN_DIR)/manager .
```

Comment:
Should we also be adding this:

```
GOOS=linux GOARCH=${ARCH}
```

Or something similar. Do we class the controller manager binary dev tooling? Probably not.

@sedefsavas @pydctw - wdyt?

Reply:
```
OS=$(go env GOOS)
GOOS=${OS} GOARCH=${ARCH}
```
@richardcase something like this should also work I believe

Reply:
That would work 👍 

(the reason in my example for hardcoding linux (`GOOS=linux`) is that we only support running the manager on linux so should we allow the manager to be built for windows or mac?)

Reply:
@richardcase In that case it would make sense to have it as `GOOS=linux GOARCH=${ARCH}` since the support is only for linux.

Reply:
That was my thought, but for `clusterawsadm` its ok to compile for windows/mac/linux etc

Reply:
Done.